src/System.Private.CoreLib/src/System/InvalidOperationException.cs

```diff
@@ -38,5 +38,12 @@ public InvalidOperationException(String message, Exception innerException)
         }
 
         protected InvalidOperationException(SerializationInfo info, StreamingContext context) : base(info, context) { }
+
+        internal static InvalidOperationException NewInvalidHandle()
```

Comment:
It's used in several places, but I noticed now that there is a SetErrorCode to set the HResult, I can move it, maybe SafeHandle is a good place for it